web/locale/en_US/messages.po

```diff
@@ -3157,3 +3157,30 @@ msgstr "Change Role"
 
 msgid "I have read and accept the <a ui-sref=\"terms-and-conditions\" target=\"_blank\"><b>Terms and Conditions here</b></a>"
 msgstr "I have read and accept the <a ui-sref=\"terms-and-conditions\" target=\"_blank\"><b>Terms and Conditions here</b></a>"
+
+msgid "Donate"
+msgstr "Donate"
+
+msgid "Maintained by"
+msgstr "Maintained by"
+
+msgid "Attend a Dojo"
+msgstr "Attend a Dojo"
+
+msgid "Volunteer"
```

Comment:
yeah the other keys are there,  but it might better, just to be sure, if you also do a quick check on the header and footer template. sorry but I would feel more confident